src/generators/operationGenerator.ts

```diff
@@ -151,45 +153,187 @@ function buildSpec(spec: OperationSpecDetails): string {
   const contentType = buildContentType(spec);
   const mediaType = buildMediaType(spec);
 
-  const isXML = spec.isXML ? "isXML: true," : "";
   const serializerName = spec.isXML
     ? "serializer: xmlSerializer"
     : "serializer";
 
-  return `{ path: "${spec.path}", httpMethod: "${
-    spec.httpMethod
-  }", responses: {${responses.join(
-    ", "
-  )}},${requestBody}${formDataParams}${queryParams}${urlParams}${headerParams}${isXML}${contentType}${mediaType}${serializerName}
-    }`;
+  writer.block(() => {
```

Comment:
Moved to a more explicit generation strategy which allows more flexibility when arranging stuff. Also although it is more verbose, it should improve readability as it is easier to follow what is happening